bcda/client/bluebutton_test.go

```diff
@@ -292,6 +292,20 @@ func (s *BBRequestTestSuite) TestGetClaim() {
 	assert.Equal(s.T(), 1, len(e.Entries))
 }
 
+func (s *BBRequestTestSuite) TestGetClaim_HashIdentifierError() {
+	existingPepper := conf.GetEnv("BB_HASH_PEPPER")
+
+	defer func() {
+		conf.SetEnv(s.T(), "BB_HASH_PEPPER", existingPepper)
+	}()
+
+	conf.SetEnv(s.T(), "BB_HASH_PEPPER", "Ã«ÃÃ¬Ã¹Ã")
```

Comment:
Are these characters deliberate to break the hash?

Reply:
yes!